### ISSUE TITLE
Generated PPTX is creating Master Slides, not Slides (as I was hoping)

### ISSUE THREAD

**Issue by richbrooks** [marp-team/marp-cli#166]
The PPTX output is awesome, but I am having an issue with 2 features.   If this is explained somewhere, can you please point me to it.

1- The generated slides are actually creating unique Master Slides and then the slide on the front end is using that Master Slide Template (for example, Slide 1 is using Master Slide 1 and Slide 2 is using Master Slide 2).   I was hoping that it would generate a regular slide.   There is probably a good reason for this.  The secondary issue with this, is if we actually apply a Master Theme to the Slide Deck, the elements of the Master Theme sit on top of the text/table/etc generated by MARP.

2.  Somewhat related to number 1 (as in this is the PPTX generated content).  The Slide contents are not editable after the fact.   It is some sort of uneditable background image.   Even on the Master Slide, I can't edit the contents.  Is there a switch or something to output editable text.  Going to reboot my machine to see if there is something just not acting right.

By the way, don't take this as negatives.  I freaking love the project.  Thank you for all of your effort.

**Comment by yhatt** [comment]
PPTX export is essentially the feature of Marp CLI. I've translated issue from Marp for VS Code ➡️ Marp CLI 🚚

1. It is caused by restriction from using JS library ([PptxGenJS](https://gitbrent.github.io/PptxGenJS/)). The rendered slide will set as background image. It's the same approach with [GitPitch](https://medium.com/@gitpitch/gitpitch-embraces-microsoft-powerpoint-keynote-google-slides-7095017dd57c) (Try to open [sample slide deck](https://1drv.ms/p/s!Ajd-AH3nCRtmc9GUVlZvLm2TQkQ)), and better for annotatation in presentation tools. But PptxGenJS has not supported setting per slide background images. Creating a lot of master slides is the only way, and we had no choice at this time.

2. Marp focuses to keep beautiful looking slides regardless of the kind of conversion. Rendering images is the best way for that. Creating editable PPTX is very hard because it's like re-creation of HTML rendering engine. Perhaps it would be much larger project than Marp.

If you want editable PPTX, we recommend to convert exported PDF into PPTX by Adobe Acrobat. However, I suppose the reproducibility of output is not good even if this. (different fonts, incorrect shadow rendering, etc...)

|Current PPTX generation<br />(Not editable / Perfect reproducibility)|PDF -> PPTX by Adobe Acrobat<br />(Editable / Incorrect renderings)|
|:---:|:---:|
|![marp-pptx](https://user-images.githubusercontent.com/3993388/65928915-bb1d3380-e43a-11e9-8129-12c182240c46.png)|![adobe-acrobat](https://user-images.githubusercontent.com/3993388/65928937-cff9c700-e43a-11e9-8cb3-245edfa55ed4.png)|



In short, now the editable PPTX with quality that Marp required cannot generate in any way.

**Comment by yhatt** [comment]
1 was improved by working in #205. A single master slide would be applied to PPTX creation in the future release.
